recipe-client-addon/package.json

```diff
@@ -26,7 +27,6 @@
     "webpack": "3.1.0"
   },
   "dependencies": {
-    "mozjexl": "1.1.5",
```

Comment:
Why did you remove this?

Reply:
I think npm did it automatically - it is a duplicate of the current line 31.